### DISCUSSION TITLE
Brave web browser like tabs

### DISCUSSION THREAD

**Discussion by samosaphile** [bromite/bromite#2272] [Ideas]
Not sure if possible but I am tired of Brave browser's push of crypto / NFTs and trying to switch browsers. However, it has one feature which is preventing Bromite from becoming my default browser of choice and that is how it handles tabs.

Here's a screenshot example of what I mean:
![Screenshot_20220818-224535](https://user-images.githubusercontent.com/30265284/185504754-f7aae883-da89-4bda-8845-caad28da2913.png)

If you look at the bottom (not the absolute bottom but the 2nd row above) there are a bunch of mini circle icons followed by a '+'. Those represent tabs and making a new tab. Pressing the '4' icon at bottom will take you to the standard chrome / Bromite window view (represents 4 windows open in background)

Anyway, it might seem like a subtle UX/GUI enhancement but to me it's huge on a 6" smartphone. It makes it so much more ergonomic switching between tabs and windows from the bottom in this style as this is close to the area where your thumb natural rests on a phone. When spending long hours surfing on a phone having to move your figure constantly towards the top to do anything tab / window related is quite fatiguing.

**Comment by uazo** [comment]
you can already enable it now because it is a chromium feature
try to set
```
chrome://flags/#enable-tab-grid-layout
chrome://flags/#enable-tab-groups
chrome://flags/#enable-tab-groups-ui-improvements
```

**Reply by samosaphile** [reply]
So it took me awhile to figure out, as I am a bit of a noob. But I finally think I figured out what you were saying after playing around with it and got it working!

I must say these settings are very finicky when making changes. For example, when doing 'enable' on a flag and relaunching browser it does nothing. But then when change back to 'default' (which seemed to be the initial setting it was set at) and relaunching browser it actually does make changes. And then when I try to like bring it back to stock bromite settings, the only thing that worked for me was 'reset all' (which I believe resets every flag to defaults). Unless I am doing something wrong? Tested on Google Pixel 4a running stock Android 12.

I also don't believe needing all these flags need to be changed. At first I made changes to all 3, but then tried just enabling 1 and it seems to do the same thing? Or maybe when you enable one it automatically makes changes to others? 

Any idea what the difference is between 'Enabled New Tab Variation' and 'Enabled New Tab Tile' ? They both seem to do the exact same thing for me. I think what I was asking/looking for was enable tab groups as that's what it's called on context menu when opening new link, 'open new tab in group' .. altho Enabling Tab Groups seems to do the exact same thing as Enabled New Tab Variation and Enabled New Tab Tile, so I'm kinda confused.

I think what would be best is if in a future bromite build this was just enabled by default for phones. Or user was prompted and given a choice between what style they prefer on initial startup / first install. (Or atleast listed as an option within bromite's settings -> 'Enable opening tab in groups')

Btw, I notice when you press and hold on a link, there is something called  'Add to reading list' on the context menu brought up. This is the default behaviour on Chrome aswell. Is there a way to remove this? I almost never use it. Just takes extra screen space. Brave doesn't seem to have it either.
